cmd/invites.go

```diff
@@ -25,9 +25,9 @@ func init() {
 				Name:  "list",
 				Usage: "List outstanding invitations for an organization. These invites have yet to be approved.",
 				Flags: []cli.Flag{
-					orgFlag("org to list invites for", true),
+					orgFlag("org to list invites for", false),
 					cli.BoolFlag{
-						Name:  "approved",
+						Name:  "all, a",
 						Usage: "Show only approved invites",
```

Comment:
This usage isn't correct anymore, is it? If so can you update it? We didn't actually update the behaviour either, it shows only approved invites. 

So, if it's going to be `--all` it should show all invites, if not, it should probably continue being `--approved`, the shortcut `-a` is 👍  either way!

Reply:
Yes, this was clumsy, misunderstood the behaviour. Have reverted it bad to "approved" but added the short "a". After taking another look, being able to show all approved vs all other types makes sense.